core/lib/dependency-graph/simulator/network-analyzer.js

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import {NetworkRequest} from '../network-request.js';
```

Comment:
Why does this type need importing everywhere now? Is it because this code will eventually live outside the LH repo?